### ISSUE TITLE
DialogHost background fade out is to slow when using MaterialDesignEmbeddedDialogHost and TransitionAssist.DisableTransitions="True"

### ISSUE THREAD

**Issue by SeynalKim** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2826]
### Bug explanation

I'd like to disable the transition of the `DialogHost` and at the same time, use the style `MaterialDesignEmbeddedDialogHost`.

```
<md:DialogHost
                Grid.Row="1"
                md:TransitionAssist.DisableTransitions="True"
                Identifier="ContainerDialogHost"
                SnackbarMessageQueue="{Binding ElementName=MySnackbar, Path=MessageQueue}"
                Style="{StaticResource MaterialDesignEmbeddedDialogHost}">

</md:DialogHost>
```

Together with `md:TransitionAssist.DisableTransitions="True"` and `MaterialDesignEmbeddedDialogHost` everything Is fast when opening the dialog. However, upon closing it, I noticed that the `Background` is slowly fading out. Are there any other settings I am missing?

### Version

using version 4.5.0

**Comment by Erapchu** [comment]
I'm just using that style

https://github.com/Erapchu/SinglePass.WPF/blob/develop/SinglePass.WPF/Themes/DialogHostTemplates.xaml

You can just remove PART_ContentCoverGrid and animations for it if you want the same effect but without that bug

**Comment by Erapchu** [comment]
https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2829

**Comment by Erapchu** [comment]
Of course I just want to leave it for future changes

**Comment by nicolaihenriksen** [comment]
Closing this issue because I believe it was included in the 4.6.0 release; see PR #2829.
